tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -42,29 +43,29 @@ def test001_create_cloudspace_with_wrong_params(self):
         self.log('%s STARTED' % self._testID)
 
         self.log('Create an cloudspace with providing non existing parameter, should fail')
```

Comment:
- an cloudspace --> a cloudspace
- Test case for creating "acloudspace" --> "a cloudspace"